feature-libs/customer-ticketing/components/customer-ticketing-reopen/customer-ticketing-reopen-dialog/customer-ticketing-reopen-dialog.component.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from '@spartacus/storefront';
 
 @Component({
-  selector: 'cx-customer-ticketing-reopen-dialog',
+  selector: 'cx-customer-ticketing-dialog',
```

Comment:
this should have not been changed. Revert back to the original distinct selector

and in your create component, put it as -create-dialog.

You can make your stylesheet similar to this https://github.com/SAP/spartacus/pull/16063/files#diff-e4ba2751cc8990d33682351a9eec88aa6247136c383656b1290abd8150c25f4aR2